easybuild/easyconfigs/d/Doxygen/Doxygen-1.8.10-GNU-4.9.3-2.25.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'CMakeMake'
+
+name = 'Doxygen'
+version = '1.8.10'
+
+homepage = 'http://www.doxygen.org'
+description = """Doxygen is a documentation system for C++, C, Java, Objective-C, Python, 
+ IDL (Corba and Microsoft flavors), Fortran, VHDL, PHP, C#, and to some extent D."""
+
+toolchain = {'name': 'GNU', 'version': '4.9.3-2.25'}
+
+sources = ['%(namelower)s-%(version)s.src.tar.gz']
+source_urls = ['http://ftp.stack.nl/pub/users/dimitri/']
+
+builddependencies = [
+    ('flex', '2.5.39'),
+    ('CMake', '3.3.2'),
+    ('Bison', '3.0.4'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/doxygen'],
```

Comment:
this is new, but good!

@hajgato: would you mind issuing a separate PR to add this to all Doxygen easyconfigs?